package/utils/busybox/Config-defaults.in

```diff
@@ -2268,6 +2268,18 @@ config BUSYBOX_DEFAULT_FEATURE_HTTPD_PROXY
 config BUSYBOX_DEFAULT_FEATURE_HTTPD_GZIP
 	bool
 	default n
+config BUSYBOX_DEFAULT_FEATURE_HTTPD_ACL_IP
+	bool
+	default n
+config BUSYBOX_DEFAULT_FEATURE_HTTPD_DATE
+	bool
+	default n
+config BUSYBOX_DEFAULT_FEATURE_HTTPD_LAST_MODIFIED
+	bool
+	default n
+config BUSYBOX_DEFAULT_FEATURE_HTTPD_ETAG
+	bool
+	default y
```

Comment:
i think you have pushed something inadvertently? these do not look related to bzcar to me...

Reply:
Omg, yes, these changes was in another branch. I’ll fix in a hour. Sorry